server/package.json

```diff
@@ -0,0 +1,39 @@
+{
+  "name": "utseus_api",
+  "version": "1.0.0",
+  "description": "",
+  "main": "app.js",
+  "engines": {
+    "node": ">=4.0.0"
```

Comment:
you must be using a more recent version than 4.0?

Reply:
You are right. 

```
 node --version
v9.2.0
```

Reply:
I rise the bar up to 9.0